### DISCUSSION TITLE
SimBrief OFP integration into the EFB

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#586] [Ideas]
This would be nice - weight and balance is already uplinked, but having the ability to view the OFP for the flight from Simbrief would be really nice, saves you having to use Avitab or something else. This is the case in other aircraft, like the the A32nx and Crj in msfs and zibo in x-plane. This would be a cool and nice feature to have.

**Comment by aeroniemi** [comment]
we do now have avitab which does this so it is kinda pointless

**Reply by Simleybob** [reply]
You need to download the pdf from simbrief and put it into the avitab charts folder - can be quite annoying. But if it’s hard to do then it’s fine. Could you make a script where it automatically downloads the pdf OFP when you press uplink load out when configuring the plane?

**Reply by aeroniemi** [reply]
we could auto-download it as we already do that for other stuff

**Reply by Simleybob** [reply]
Epic. Could you make that an external plugin as well if possible to use with other aircraft?

**Reply by aeroniemi** [reply]
if you really wanted it to be, i guess
